round4/group16

```diff
@@ -0,0 +1,60 @@
+Chrome Extension Idea: AI Research Assistant
+Authors
+Runzhi Gu, Sam Bhatta, Daniel Kwon, Caroline Begg
+
+Problem Statement
+Students, researchers, and information-seekers often struggle with the overwhelming task of sifting through vast amounts of online content to find relevant information. There is a lack of effective tools that are embedded in the web browser that helps the users quickly understand a long and complicated web article, which makes the research process daunting and less productive.
+
+Target Audience
+Students conducting research for academic papers or projects. Researchers gathering up to date information across various fields. Educators and content creators that collect rephrased information. Anyone who frequently engages in extensive online reading and needs to compile information efficiently. 
+
+Description
+The AI Research Assistant is a Chrome Extension powered by Open AI api through the GPT model, designed to enhance the online research process. It offers a sidebar interface that allows users to summarize web pages or specific sections, and displays it in an organized and refined manner. 
+
+Selling Points
+Efficient Information Processing: The extension swiftly summarizes long and complex web articles, condensing them into digestible highlights. This drastically reduces the time required for users to sift through and understand vast amounts of online content, making the research process more manageable and less overwhelming.
+Tailored Summarization: It provides users with the flexibility to select specific sections of a webpage for summarization. This feature ensures that users can focus on extracting information that is most relevant to their research needs, without having to wade through unrelated content.
+User-centric Design: The focus on a simple, intuitive user experience ensures that the tool is accessible to users with varying levels of technical expertise. Integrates directly into the web browser with a sidebar interface, providing a seamless tool for gathering, understanding, and organizing information without switching between apps.
```

Comment:
I like how you're centering your design around the user. A suggestion that could slightly mprove the quality of life of users of your extension is to allow them to either highlight text, as you mentioned, or allow them to perhaps switch to a "select mode", where instead of highlighting a long paragraph (in which they may have to scroll and accidentally misclick and have to redo the highlighting), they can simply click on a paragraph or sentence, and your extension will do the work on retrieving the entire paragraph/sentence.